pkg/common/ipsec/utils.go

```
@@ -30,9 +31,9 @@ func CountUniqueIPsecKeys(states []netlink.XfrmState) (int, error) {
 			keys[key] = true
 			continue
 		}
-		return 0, fmt.Errorf("an unsupported XfrmStateAlgo combination has been found")
+		_, _ = fmt.Fprintf(os.Stderr, "an unsupported XfrmStateAlgo combination has been found\n")
```

Nit: it seems that `_, _ =` is not really needed, at least all the other occurrences of Fprintf don't need it.

It's also a good idea to print it only once, because now it will output a line per an unsupported xfrm state.

---

Good catch about duplicated outputs, fixed.

Regarding `_, _ =`, it is usually a good practice to ignore errors explicitly (e.g.: my IDE complains about it) but its not required. Removed.